packages/jsActions/native-mobile-resources/package.json

```diff
@@ -17,13 +20,14 @@
     "@react-native-community/push-notification-ios": "^1.5.0",
     "@react-native-firebase/messaging": "10.5.1",
     "react-native-device-info": "8.0.1",
-    "react-native-image-picker": "2.3.1",
+    "react-native-image-picker": "4.0.3",
```

Comment:
Missing main package-lock change

Reply:
The p-lock has never been tracked/committed

`git log -p --follow -- ~/mendix/widgets-resources/packages/jsActions/native-mobile-resources/package-lock.json` returns no history

Should I commit the file?

Reply:
The only one that should be updated is the main package-lock :)

Reply:
Unless you by mistake did npm install inside the package it will never generate a package-lock individually for each package.

Reply:
This is the mistake I made, whoops! Corrected by `npm i` in root. Thanks!